swagger_server/scripts/update_ids.py

```diff
@@ -0,0 +1,23 @@
+import pickle
+import sys
+
+from py2neo import Graph
+
+from swagger_server.exceptions import NotFound
+from swagger_server.ogm.mappers import SampleNode
+
+if __name__ == '__main__':
```

Comment:
Even though it is simpler to update the ids via accessing the database directly, there could be problems in the future such as access issue if the database is restricted etc. Accessing and updating the data through the API is preferable, imho. Also this is quite specific to our current setup, so it would be better to have a script in a separate repo.

The rest of this pull request on the dev changes are good so we can still keep those.